docs/internals/network/basic_networking/basic_networking.md

```diff
@@ -0,0 +1,250 @@
+# Basic Networking
+
+In this part we will go through some of basic Networking Concepts that is very related to how networking in zos working
+
+## Components
+
+- Namespaces: As you may know linux have the concept of Namespaces which includes (users, net, ....) namespaces. which is basically considered as a full copy of the network stack that is fully isolated from the network stack of the machine
+- Bridge: A network bridge is a virtual network device that forwards packets between two or more network segments. A bridge behaves like a virtual network switch and works transparently.
```

Comment:
Between network segements (remove two or more)

Reply:
I copied it originally from here https://wiki.archlinux.org/title/network_bridge added link on Bridges: so ppl can check it for more info